migrations/20200519000443-initial-table-creation.js

```diff
@@ -0,0 +1,85 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.createTable('users', { id: Sequelize.INTEGER });
+    */
+    await queryInterface.createTable('authors', {
+      id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
+      nameFirst: { type: Sequelize.STRING, allowNull: false },
+      nameLast: { type: Sequelize.STRING, allowNull: false },
+      createdAt: { type: Sequelize.DATE, defaultValue: Sequelize.literal('CURRENT_TIMESTAMP') },
+      updatedAt: {
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.literal('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP')
+      },
+      deletedAt: { type: Sequelize.DATE },
+    })
+
+    await queryInterface.createTable('genres', {
+      id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
+      genre: { type: Sequelize.STRING, allowNull: false },
```

Comment:
This is weird naming since a `findOne` would return a single genre so then your code would have to use `genre.genre` to get at this value. Something like `genre.name` is better.